docs/src/.vuepress/theme.ts

```diff
@@ -93,7 +93,7 @@ export default hopeTheme({
       {
         text: "Security",
         icon: "safe",
-        link: "/security",
+        link: "security/",
```

Comment:
seems incorrect, the removal of the first character makes the link unusable in different directories